.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will upload a Python Package using flit when a release is
+# created.  For more information see:
+# https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install wheel twine setuptools
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: __token__
+        # The PYPI_API_TOKEN must be a pypi token with the "pypi-" prefix with
+        # sufficient permissions to upload this package:
+        # https://pypi.org/help/#apitoken
+        TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
This token is in the repo secrets:
![image](https://user-images.githubusercontent.com/13209176/167663992-e1b197de-9767-4cda-93ad-b2a46ec3e701.png)